src/test/java/org/jsoup/nodes/ElementTest.java

```diff
@@ -1215,19 +1215,19 @@ public void testAppendTo() {
 
     @Test public void testNormalizesInvisiblesInText() {
         // return Character.getType(c) == 16 && (c == 8203 || c == 8204 || c == 8205 || c == 173);
```

Comment:
Could just remove this comment but....

```suggestion
        // return Character.getType(c) == 16 && (c == 8203 || c == 8205 || c == 173);
```

Reply:
Good spot! Thanks!